ONSdigital/design-system
# Base page template config enhancements

### What is the context of this PR?
Fixes #2095 

Also allows control of the container for all components in template by setting `pageConfig.wide` or `pageConfig.fullWidth` just once, rather than within each object.

### How to review
- Check page template config header options work as expected
- Check docs
- Check updated tests
